crates/sui-indexer/src/handlers/committer.rs

```diff
@@ -71,7 +71,7 @@ where
                 })?;
             }
         }
-        if !batch.is_empty() && unprocessed.is_empty() {
```

Comment:
Curious, would `unprocessed.is_empty()` always be true here? Since in the while look above, we loop until `unprocessed` is empty?

Reply:
not really, checkpoints out of the stream is not in order, let's say we have cp 10, 11, 13, here 13 is in unprocessed, this pr intends to commit 10 and 11 without waiting for 12

Reply:
Got it. Thanks!